Sources/NIOSSL/SSLErrors.swift

```diff
@@ -82,6 +82,8 @@ extension NIOSSLError: Equatable {
     }
 }
 
+public struct NIOSSLCloseTimedOutError: Error {}
```

Comment:
Can we have a doc comment for this please?

Reply:
@Lukasa I told George that he might get away without one given that none of the errors above have a comment ;). But maybe this one warrants a bit more explanation.

Reply:
Done!